### PR TITLE
Add flatcar image requirements for getting started with Azure 

### PR DESCRIPTION
**Pull request by T-Kukawka**
### Please remember to

- [x] Give the PR a meaningful title -- it will be shown in the release notes
- [x] Add (or remove) [user questions](https://github.com/giantswarm/docs/blob/master/CONTRIBUTING.md#front-matter) associated with any updated docs

### CODE REVIEW

File: src/content/getting-started/cloud-provider-accounts/azure/index.md [hunk 1]
```diff
@@ -126,6 +127,17 @@ Please remember to change the `roleDefinitionId` in case you would like to use y
...
+## Accept legal terms for deeployment of Flatcar image
+
+Giant Swarm deploys Flatcar image developed by Kinvolk from Azure Marketplace. In order to be able to run the image, it is required by Azure to accept the legal terms.
+Please run following command prior to launching a cluster on a given subscription:
+
+```nohighlight
+az vm image terms accept --offer flatcar-container-linux-free --plan stable --publisher kinvolk
+```
+
+This acceptance should be performed for all subscriptions that are usedd to run Giant Swarm workload clusters.
```
Comment (fiunchinho) [inline src/content/getting-started/cloud-provider-accounts/azure/index.md:139]:
```suggestion
This acceptance should be performed for all subscriptions that are used to run Giant Swarm workload clusters.
```

File: src/content/getting-started/cloud-provider-accounts/azure/index.md [hunk 2]
```diff
@@ -126,6 +127,17 @@ Please remember to change the `roleDefinitionId` in case you would like to use y
 
 This command should be run for all subscriptions that are used for Giant Swarm workload clusters as well as the management cluster that orchestrates it all.  
 
+## Accept legal terms for deeployment of Flatcar image
+
+Giant Swarm deploys Flatcar image developed by Kinvolk from Azure Marketplace. In order to be able to run the image, it is required by Azure to accept the legal terms.
```
Comment (fiunchinho) [inline src/content/getting-started/cloud-provider-accounts/azure/index.md:132]: should we link somewhere? like to the marketplace or something?

Comment (T-Kukawka) [reply src/content/getting-started/cloud-provider-accounts/azure/index.md:132]: yeah just had same thought reading this

File: src/content/getting-started/cloud-provider-accounts/azure/index.md [hunk 3]
```diff
@@ -126,6 +127,17 @@ Please remember to change the `roleDefinitionId` in case you would like to use y
...
+Please run following command prior to launching a cluster on a given subscription:
```
Comment (fiunchinho) [inline src/content/getting-started/cloud-provider-accounts/azure/index.md:133]:
```suggestion
Please run the following command prior to creating a cluster on a given subscription:
```

File: src/content/getting-started/cloud-provider-accounts/azure/index.md (hunk 2 above)
Comment (T-Kukawka) [reply src/content/getting-started/cloud-provider-accounts/azure/index.md]: Done

**Review by fiunchinho — APPROVED** [review]

**Pull request merged**